contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,88 @@
+
+### Major Change:
+- If the PR is a major change which may affect a large section of your repository, you can pull the local changes of the PR into your local system and browse through each and every change made in the PR. You can also test & play around with the changes done in the PR to ensure each and every change is as expected.
+- To fetch the Pull Request locally, use this -
+```sh
+$ git checkout -b `<pr-to-review>`
+$ git pull upstream/<PULL ID>/head
+```
+
+## Reviewing Ideology
+
+### Deciding what to review ?
+- As a Maintainer, you might be completely overloaded with the Pending PRs left to review, some of which will be from experienced contributors and some from a new contributor.
+- Since you know the experienced contributor already knows the guidelines for PR and Code to be written for repository, you should review the PR and patches from the experienced contributors first, as it will be less time consuming.
+- Until you have a large block of free time, you must put the newcomer PR on hold. Afterwhich, you should go line by line through the code changes in the PR - and please avoid criticizing  the `Code Structure`, `Coding Style` of the newcomer's PR.
+
+### Assigned Reviews
+- Maintainers can assign reviews to other maintainers, when appropriate. The
+assignee becomes the shepherd for that PR and is responsible for merging the PR
+once they are satisfied with it or else closing it. The assignee might request
+reviews from non-maintainers.
+
+### What to approve ?
+- Before approving the `Patch/PR`, Make sure :
+    + If it is a bug fix:
+        * Does the following actually fix the bug ?
+        * Is the bug real/worth fixing ?
+        * Is this the right fix ?
+    + If it is a feature request:
+        * Is the community okay with this feature ?
+        * Do we really need this feature to move forward?
+        * Does it break the other features in the repository?
+- If the scope of change is very large, it is always better to ask for incremental change PRs as they are much more easy to interact and review.
+- Make sure the changes contain comments :
+    + If the patch does *anything* non-obvious, it needs to explain why and how with proper comments.
+    + If you made any decision while making some analysis, explain why you chose the following and on what basis.
+- Final note to reviewers, code review time is exponential to the PR size, hence it's better to check each change line by line so that it does not affect any other feature in the repository.
+
+### What happens after successful review ?
+- If the Pull Request looks good, you can trigger the PR Builder to test the PR for tests defined.
+    + PR Builder can be triggered by a reply of `/ok-to-test` by a **Kubernetes Member**
+- A PR is considered to be `Ready To Merge` if it matches the `Tide Conditions` after which `Tide` can successfully merge the PR. You can find the set of conditions [here](https://prow.k8s.io/tide).
```

Comment:
```suggestion
- A PR is considered to be `Ready To Merge` once it meets the [_merge criteria_](http://git.k8s.io/test-infra/prow/cmd/tide/pr-authors.md). After which [Tide](https://prow.k8s.io/tide) can batch the PR to be merged.
```
I'd hold off on introducing a new term here. The tide docs refer to the conditions as _merge_criteria_. ref: https://git.k8s.io/test-infra/prow/cmd/README.md#merge-automation

Using the same terminology will help folks when looking stuff up 👍